feature/qos/otg_tests/ingress_police_nhg/README.md

```diff
@@ -0,0 +1,363 @@
+# TE-18.2 QoS scheduler with 1 rate 2 color policer, classifying on next-hop group
+
+## Summary
+
+Use the gRIBI applied ip entries from TE-18.1 gRIBI. Configure an ingress scheduler
+to police traffic using a 1 rate, 2 color policer. Configure a classifier to match
+traffic on a next-hop-group.  Apply the configuration to a VLAN on an aggregate
+interface.  Send traffic to validate the policer.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+Use TE-18.1 test environment setup.
+
+## Procedure
+
+### TE-18.2.1 Generate and push configuration
+
+* Generate config for 2 classifiers which match on next-hop-group.
+* Generate config for 2 forwarding-groups mapped to "dummy" input queues
+  * Note that the DUT is not required to have an input queue, the dummy queue
+    satisfies the OC schema which requires defining nodes mapping
+    classfier->fwd_group->queue->scheduler
+* Generate config for 2 schedulder-policies to police traffic
+* Generate config to apply classifer and scheduler to DUT subinterface with vlan.
+* Use gnmi.Replace to push the config to the DUT.
+
+```yaml
+---
+openconfig-qos:
+  classifers:
+    - classifer: “dest_A”
+      config:
+        name: “dest_A”
+      terms:
+        - term:
+          config:
+            id: "match_1_dest_A1"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_A1"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+          actions:
+            config:
+              target-group: "input_dest_A"
+        - term:
+          config:
+            id: "match_1_dest_A2"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_A2"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+          actions:
+            config:
+              target-group: "input_dest_A"
+
+    - classifer: “dest_B”
+      config:
+        name: “dest_B”
+      terms:
+        - term:
+          config:
+            id: "match_1_dest_B1"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_B1"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+          actions:
+            config:
+              target-group: "input_dest_B"
+        - term:
+          config:
+            id: "match_1_dest_B2"
+          conditions:
+            next-hop-group:
+                config:
+                    name: "nhg_B2"     # new OC path needed, string related to /afts/next-hop-groups/next-hop-group/state/next-hop-group-id
+          actions:
+            config:
+              target-group: "input_dest_B"
+
+
+  forwarding-groups:
```

Comment:
Kindly add more context. I do not understand what is this section delivering. Thank you

Reply:
I have updated the OC qos overview documentation to include an in depth explanation of ingress policing.  Please see https://github.com/openconfig/public/pull/1190/files?short_path=11f0b86#diff-11f0b8695aa64acdd535b0d47141c0a373e01f63099a423a21f61a542eda0052